cli/src/main.rs

```diff
@@ -155,6 +156,38 @@ fn build_app<'a, 'b>() -> App<'a, 'b> {
                     )
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("find-path")
+                 .about("Fetch possible payment paths")
+                            .arg(
+                                Arg::with_name("destination_account")
+                                    .long("destination_account")
+                                    .takes_value(true)
+                                    .required(true)
+                                    .help("The destination account id of the payment path to query"),
+                            )
+                            .arg(
+                                Arg::with_name("source_account")
+                                    .long("source_account")
+                                    .takes_value(true)
+                                    .required(true)
+                                    .help("The source account id of the payment path to query"),
+                            )
+                            .arg(
+                                Arg::with_name("destination_asset")
```

Comment:
There's only one asset that needs to be specified. It might make sense to label the action `asset` instead of making it longer.

In fact, you may think about how we can make this easy to use from the command line. Long argument names are a pain to type out and so it might make sense to make them shorter (the name of the arg can stay the same):

```
stellar find-path --from=<source account id> --to=<destination account id> --asset=XLM --amount=1.123456
```